source/gui/guiHelper.py

```diff
@@ -124,12 +124,30 @@ def addButton(self, *args, **kwargs):
 		return wxButton
 
 
-def associateElements(firstElement: wx.Control, secondElement: wx.Control) -> wx.BoxSizer:
+# vertical controls where the label should go above visually, and the control should go below
+_VerticalCtrlT = TypeVar("_VerticalCtrlT", wx.ListCtrl, wx.ListBox, wx.TreeCtrl)
+# horizontal controls where the label should go first visually, and the control should go after
+_HorizontalCtrlT = TypeVar(
+	"_HorizontalCtrlT",
+	wx.Button,
+	wx.Choice,
+	wx.ComboBox,
+	wx.Slider,
+	wx.SpinCtrl,
+	wx.TextCtrl,
+)
+
+
+def associateElements(
+	firstElement: wx.StaticText | wx.StaticText | wx.StaticText | wx.Button | wx.TextCtrl,
+	secondElement: _HorizontalCtrlT | wx.CheckBox | _VerticalCtrlT | wx.CheckBox | wx.Button,
+) -> wx.BoxSizer:
 	"""Associates two GUI elements together. Handles choosing a layout and appropriate spacing. Abstracts away common
 	pairings used in the NVDA GUI.
 	Currently handles:
-		wx.StaticText and (wx.Choice, wx.TextCtrl, wx.Slider, wx.Button or wx.SpinCtrl) - Horizontal layout
-		wx.StaticText and (wx.ListCtrl or wx.ListBox or wx.TreeCtrl ) - Vertical layout
+		wx.StaticText and _HorizontalCtrlT - Horizontal layout
```

Comment:
Can we either spell out control types, or explicitly link to the typevar here?

Reply:
I think that might be worse, as then the documentation has a risk of getting out of sync again to the actual parameters. We try to avoid writing types in docstrings for this reason.

Reply:
Could you clarify what you mean by explicitly link?

Reply:
```suggestion
		wx.StaticText and :const:`_HorizontalCtrlT` - Horizontal layout
```
My primary concern here is that the horizontal and vertical control types are private variables, so if/when we get to building API docs, they won't be represented in the docs.